packages/reports/addon/components/filter-values/range-input.ts

```diff
@@ -35,8 +32,8 @@ export default class RangeInput extends Component {
    */
   @action
   setLowValue({ target: { value } }) {
-    this.onUpdateFilter({
-      values: [value, get(this, 'filter.values.lastObject')]
+    this.args.onUpdateFilter({
+      values: [value, this.args.filter.values[0]]
```

Comment:
```suggestion
      values: [value, this.args.filter.values[1]]
```
Is this right? We want to preserve the high value and change the low value